app.rb

```diff
@@ -0,0 +1,112 @@
+require_relative 'teacher'
+require_relative 'rental'
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+
+class App
+  def run
+    puts 'Welcome to School Library App!'
+    loop do
+      puts '-' * 50
+      puts '
+          1- List all books
+          2- List all people
+          3- Create a new person
+          4- Create a new book entry
+          5- Create a new rental entry
+          6- List all rentals for a given person id
+          7- Quit'
+      puts 'Choose an option: '
+      input = gets.chomp.to_i
+
+      break if input == 7
+
+      operation(input)
+    end
+  end
+
+  def operation(input)
+    case input
+    when 1
+      list_books
+    when 2
+      list_people
```

Comment:
- [ ] For user inputs 1 and 2, You are calling methods `list_books`, and `list_people`, But there are no such methods on this file. Please implement these methods and list books and people inside the methods.